conf/application.conf

```diff
@@ -52,6 +52,7 @@ play.ws.timeout.request = 15000ms #15 seconds
 # Lost Creds, VAT Reg and IV can call identity-verification-questions and utilise the questions within it for now
 allowedUserAgentList=["identity-verification", "lost-credentials", "lost-credentials-frontend", "nino-identity-verification", "govuk-one-login"]
 
+deniedUserAgentListForP45=["identity-verification", "lost-credentials", "lost-credentials-frontend", "govuk-one-login"]
```

Comment:
It would be good to log the active denied user agent list somewhere to confirm the configuration.